src/Sarif/PerLanguageOption.cs

```diff
@@ -51,10 +51,28 @@ public PerLanguageOption(string feature, string name, Func<T> defaultValue, stri
                 throw new ArgumentException(nameof(name));
             }
 
-            this.Feature = feature;
-            this.Name = name;
-            this.DefaultValue = defaultValue;
-            this.Description = description;
+            Feature = feature;
+            Name = name;
+            DefaultValue = defaultValue;
+            Description = null;
+        }
+
```

Comment:
Possibility of creating a default class, but that overrides the getter and setter methods. Will research better way to do this.